lib/src/upgrade_messages.dart

```diff
@@ -153,7 +153,7 @@ class UpgraderMessages {
         break;
       case 'hi':
         message =
-            '{app name} का एक नया संस्करण उपलब्ध है। संस्करण {{currentAppStoreVersion}} अब उपलब्ध है-आपके पास है {{currentInstalledVersion}}.';
+            '{{app name}} का एक नया संस्करण उपलब्ध है। आपके पास संस्करण {{currentInstalledVersion}} है, लेकिन अब {{currentAppStoreVersion}} उपलब्ध है।';
```

Comment:
Thanks for the PR. I like the change for the incorrect braces around app name. However, I do not like the other changes for the language text. The intent of these messages has been changed, not just simple syntax. I don't want to accept these changes unless I hear from some other Hindi speakers.

Reply:
I fixed the missing braces for app name in the code so there is no need for this PR now.

Reply:
@aruhant I would like to better understand the proposed changes first, which seem to change the meaning of the existing text.

Reply:
Alright, no problem! Those proposed changes merely use precise, more appropriate wording and do not alter the intended meaning. Though, feel free to double-check with other Hindi speakers. I am happy to clarify anything that may be unclear.